src/plugins/dam/components/asset-screen/ImageAssetFields.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Media } from 'reactstrap';
+import damUrl from 'plugins/dam/damUrl';
+
+export default function ImageAssetFields({asset}) {
+  return (
+      <Media
```

Comment:
@meghedi could render dimensions here (it's not editable so really doesn't need to be field)

Reply:
@gdbrown moved dimension function to ImageAssetFields.jsx, and its a readonly field.